pgwatch3_rpc_server/src/examples/csv_receiver.go

```diff
@@ -0,0 +1,87 @@
+package examples
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	. "pgwatch3_rpc_receiver/sinks"
```

Comment:
Dot-import should be used only in extreme cases, like testing. Are there any particular reason to use dot-import here? Why not just regular `sinks` package?

Reply:
This was just for testing, I'll remove this.

Reply:
Ok, for some reason removing dot imports are failing. I'll keep them for now, will mark this in a separate issue.